modules/axdriver/src/drivers.rs

```diff
@@ -66,3 +66,51 @@ cfg_if::cfg_if! {
         }
     }
 }
+
+cfg_if::cfg_if! {
+    if #[cfg(net_dev = "ixgbe")] {
+        use crate::ixgbe::IxgbehalImpl;
+        use axhal::mem::phys_to_virt;
+        pub struct IxgbeDriver;
+        register_net_driver!(IxgbeDriver, driver_net::IxgbeNic<IxgbehalImpl, 1>);
+        impl DriverProbe for IxgbeDriver {
+            fn probe_pci(
+                    root: &mut driver_pci::PciRoot,
+                    bdf: driver_pci::DeviceFunction,
+                    dev_info: &driver_pci::DeviceFunctionInfo,
+                ) -> Option<crate::AxDeviceEnum> {
+                    use crate::ixgbe::IxgbehalImpl;
+                    use driver_net::{INTEL_82599, INTEL_VEND, IxgbeNic};
+                    if dev_info.vendor_id == INTEL_VEND && dev_info.device_id == INTEL_82599 {
+                        // Intel 10Gb Network
+                        info!("ixgbe PCI device found at {:?}", bdf);
+
+                        // Initialize the device
+                        // These can be changed according to the requirments specified in the ixgbe init function.
+                        const QS: u16 = 1;
```

Comment:
Why queue size is only 1?

Reply:
Sorry, this is my mistake. Here, `QS` actually means the number of network card queues, which may be ambiguous.